content/docs/04.5-devx/05-app-profile/15-db-services.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Database Services"
+metaTitle: "Palette Dev Engine Database Services"
+metaDescription: "Explore Palette Dev Engine Database Services"
+icon: ""
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# Overview
+
+Palette Dev Engine facilitates database service setup, operation, and scaling without installing physical hardware, software, or performance configurations. Instead, Palette takes care of all the administrative and maintenance tasks so that you can use and access the database quickly. Palette also offers control preferences based on the database service provider offerings and your preferences.
```

Comment:
```suggestion
Palette Dev Engine facilitates database service setup, operation, and scaling without installing physical hardware, software, or performance configurations. Instead, Palette takes care of all the administrative and maintenance tasks so that you can use and access the database quickly. 
```
Let's remove the later part as the control is limited in the current offering.